src/v2i-hub/PortDrayagePlugin/src/WebServiceClient.cpp

```diff
@@ -2,274 +2,267 @@
 
 
 WebServiceClient::WebServiceClient() {
-    // Set polling frequency to 5s
-    polling_frequency = 5;
-    // Initialize API
-    api = new OAIDefaultApi(0);
-    // Setup Server config
-    api->setNewServerForAllOperations(
-        QUrl("http://127.0.0.1:8090"),
-        QString::fromStdString("Unsecured hosting for development"),
-        QMap<QString, OAIServerVariable>()
-    );
+    initialize("127.0.0.1", 8090, false, 5);
```

Comment:
Missing file header.  I noticed that the header information in PortDrayagePlugin.cpp is not consistent with other plugins, please check and let's discuss.

Reply:
Not sure what you are referring to